### ISSUE TITLE
When Running a command with expired token, client-id is missing on the session.

### ISSUE THREAD

**Issue by gcarrarom** [gcarrarom/fancy-abak#2]
When running the first command with an expired token, fancy-abak will issue a new authentication command; when that happens in a command that uses the context of client or project ID, the following error happens:
```
Usage: abak timesheet set [OPTIONS]
Try 'abak timesheet set --help' for help.

Error: Invalid value: Oh, hey there! Sorry bud, but client id cannot be empty!
```

This is probably due to the authentication command happening before the set of context, and changing the flow of the command.

**Comment by gcarrarom** [comment]
Should be fixed by #7

**Comment by gcarrarom** [comment]
Looks good!
